readme.md

```diff
@@ -94,6 +196,28 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Your Answer:**
 
+It gets called every time there's a request to the server.
+
+Output: (see 1st or 2nd line)
+
+```Listening on Port 5000
+In the server!
+{
+  host: 'localhost:5000',
+  connection: 'keep-alive',
+  pragma: 'no-cache',
+  'cache-control': 'no-cache',
+  'upgrade-insecure-requests': '1',
+  'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) ' +
+    'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 ' +
+    'Safari/537.36',
+  accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3',
+  'accept-encoding': 'gzip, deflate, br',
+  'accept-language': 'en-US,en;q=0.9'
+}
+{}
+```
+
```

Comment:
Remember app.use gets called every time a request makes it to that point of a route chain.